e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -1059,6 +1059,93 @@ describe("scenarios > embedding > dashboard appearance", () => {
 
     H.main().should("have.css", "font-family", "Roboto, sans-serif");
   });
+
+  it("should use transparent pivot table cells in static embedding's dark mode (metabase#61741)", () => {
+    const testQuery = {
+      type: "query",
+      query: {
+        "source-table": ORDERS_ID,
+        aggregation: [["count"]],
+        breakout: [
+          [
+            "field",
+            PEOPLE.SOURCE,
+            { "base-type": "type/Text", "source-field": ORDERS.USER_ID },
+          ],
+          [
+            "field",
+            PRODUCTS.CATEGORY,
+            { "base-type": "type/Text", "source-field": ORDERS.PRODUCT_ID },
+          ],
+        ],
+      },
+      database: 1,
+    };
+
+    const pivotQuestionDetails = {
+      name: "Pivot Table Test",
+      query: testQuery.query,
+      display: "pivot",
+    };
+
+    const pivotDashboardDetails = {
+      name: "Pivot Dashboard Test",
+      enable_embedding: true,
+      embedding_params: {},
+    };
+
+    H.createQuestionAndDashboard({
+      questionDetails: pivotQuestionDetails,
+      dashboardDetails: pivotDashboardDetails,
+    }).then(({ body: { dashboard_id } }) => {
```

Comment:
The bug only happens in pivot table cells in dashboards. Does not happen in questions.